mapper/thin_edge_json/src/serialize.rs

```diff
@@ -63,13 +65,30 @@ impl ThinEdgeJsonSerializer {
             }
         }
 
-        self.buffer.push(b'}');
+        self.buffer.push('}');
         Ok(())
     }
 
     pub fn bytes(mut self) -> Result<Vec<u8>, ThinEdgeJsonSerializationError> {
         self.end()?;
-        Ok(self.buffer)
+        Ok(self.buffer.into())
+    }
+
+    // XXX: We need to abstract all this into a JsonSerializer.
```

Comment:
I fully agree with the proposal done in https://github.com/thin-edge/thin-edge.io/pull/278